securityscanutils/securityscan.go

```diff
@@ -248,8 +266,14 @@ func (r *SecurityScanRepo) RunMarkdownScan(ctx context.Context, release *github.
 		}
 
 	}
+	localOutputFilename := ""
+	if r.Opts.OutputResultLocally {
+		githubIssueOutputDir := path.Join(r.Opts.OutputDir, r.Repo, "github_issue_results")
+		err = os.MkdirAll(githubIssueOutputDir, os.ModePerm)
```

Comment:
These two lines seem to be responsible for setting up the output directory, and I imagine they only need to be executed once. At the moment, they are executed for each version that is scanned, which means that if we scan 100 images (often more), we will execute this 100 times instead of just once.

I think we can move this setup logic high in the function to just run once, what do you think?